pandas/io/gbq.py

```diff
@@ -578,10 +617,14 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
     https://developers.google.com/api-client-library/python/apis/bigquery/v2
 
     Authentication to the Google BigQuery service is via OAuth 2.0.
-    By default user account credentials are used. You will be asked to
-    grant permissions for product name 'pandas GBQ'. It is also posible
-    to authenticate via service account credentials by using
-    private_key parameter.
+    If "private_key" is not provided:
```

Comment:
use ``-`` here to make items.

Add in a versionadded tag (for the changed one)